### DISCUSSION TITLE
Formatting Loki timestamps in Grafana

### DISCUSSION THREAD

**Discussion by nkov** [grafana/grafana#52974] [Observability]
**Is your feature request related to a problem? Please describe.**
The Logs panel of Loki displays timestamps as `%Y-%m-%d %H:%M:%S`. Sometimes it would be nice to see a relative time.

**Describe the solution you'd like**
A way to configure the time format in the Logs Panel, eg. "Date From" format ("30 minutes ago"), or an option in `line_format`. This is the desired outpt:
```
(30 minutes ago) User A performed an action
(35 minutes ago) User B performed an action
(2 days ago) User A performed an action
```

**Describe alternatives you've considered**
Table with field override but couldn't figure out how to make this work with Loki

**Additional context**
![image](https://user-images.githubusercontent.com/3165749/132816739-89265ba3-3bb1-4b36-8acd-acd8852a8367.png)

**Comment by dannykopping** [comment]
Thank you for your feature request.

I'm going to move this to the `grafana/grafana` project as `grafana/loki` is just for the Loki database product.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
